docs/template-gcp-appengine.md

```diff
@@ -0,0 +1,40 @@
+The SAFE template has the ability to deploy to [AppEngine](https://cloud.google.com/appengine/) on Google Cloud Platform, GCP, with minimum effort. The deployment relies on AppEngine's flex deployment, that is, it is deploying a docker container.
+
+## Quickstart
+
+If you are familiar with GCP and already have `gcloud` configured all you need to do to get started is:
+
+```
+dotnet new SAFE --deploy gcp-appengine
+fake run build.fsx -t Deploy
```

Comment:
change to `fake build -t Deploy`

Reply:
Ah, didn’t know about that convention. Will fix.

Reply:
Ah, didn’t know about that convention. Will fix.